### PR TITLE
Add `upgrade-support` link on issue creation

### PR DESCRIPTION
**Pull request by lucasbento**
## Summary

This PR adds a https://github.com/react-native-community/upgrade-support link for the user when creating an issue.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -3,6 +3,9 @@ contact_links:
     - name: 📃 Documentation Issue
       url: https://github.com/facebook/react-native-website/issues
       about: Please report documentation issues in the React Native website repository.
+    - name: ⤴️ Upgrade Issue
+      url: https://github.com/react-native-community/upgrade-support
+      about: Please report upgrade issues in Upgrade Support repository.
```
Comment (alloy) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
      about: Please report upgrade issues in the Upgrade Support repository.
```

**Comment by lucasbento** [comment]
Awesome, thanks @alloy!

**Comment by hramos** [comment]
How about something like “Need help upgrading to a newer React Native release? Visit the Upgrade Support repository.” or something along those lines? We should assume that folks going through this flow will not have read the blog post that introduces this.

**Comment by lucasbento** [comment]
@hramos: makes sense, just updated it.

**Pull request closed without merging**